plugins/uefi-capsule/README.md

```diff
@@ -183,6 +183,13 @@ Modify `BootOrder` as well as `BootNext` to work around BIOS bugs.
 
 Use Dell customized file location for the capsule on disk.
 
+### `Flags=no-extra-esp-space`
+
+The UEFI ESRT entry does not need extra ESP space to write a backup ROM or for logging.
+If this flag is set then the `RequireESPFreeSpace` config file option is ignored.
+
+Since: 2.0.6
+
```

Comment:
how about instead of a quirk to say no extra - the quirk can set _exactly_ how much is needed?

Reply:
I thought about hooking up `device_class->set_quirk_kv` -- but the metadata can only set specific flags -- and not random device properties.

Reply:
i.e. we've got FU_DEVICE_PRIVATE_FLAG_MD_SET_FLAGS, SET_NAME, SET_SIGNED and that's probably as much as I'm comfortable leaking out of the metadata....

Reply:
Ok. Then this is practical.